test/test_utils/__init__.py

```diff
@@ -100,6 +100,9 @@
 
 SAGEMAKER_EXECUTION_REGIONS = ["us-west-2", "us-east-1", "eu-west-1"]
 
+UPGRADE_ECR_REPO_NAME = f"""upgraded-image-repo-{boto3.client("sts").get_caller_identity().get("Account")}"""
```

Comment:
An ECR repository is an account-specific resource, and therefore doesn't need the account ID to be a part of its name. We only need the account ID to be in the name of S3 buckets because S3 buckets are global resources across all AWS accounts.
```suggestion
UPGRADE_ECR_REPO_NAME = "upgraded-image-ecr-scan-repo"
```